app/renderer/components/main/braveryPanel.js

```diff
@@ -262,55 +271,63 @@ class BraveryPanel extends ImmutableComponent {
           compactBraveryPanel && styles.braveryPanel_compact__stats
         )}>
           <div data-test-id='adsBlockedStat'
-            onClick={this.onToggleAdsAndTracking}
             className={css(
               styles.braveryPanel__stats__item_count_adsBlockedStat,
-              !!adsBlockedStat && styles.braveryPanel__stats__item_count_clickable,
               (!shieldsUp || adControl === 'allowAdsAndTracking') && styles.braveryPanel__stats__item_count_disabled,
               gridStyles.row1col1,
               !compactBraveryPanel && styles.braveryPanel__stats__item,
               !compactBraveryPanel && styles.braveryPanel__stats__item_count,
               compactBraveryPanel && styles.braveryPanel_compact__stats__item_count
-            )}>{adsBlockedStat}</div>
+            )}>
+            <span onClick={this.onToggleAdsAndTracking}
```

Comment:
based on above, test id should belong to here, i.e. `<span data-test-id='adsBlockedStat' onClick={this.onToggleAdsAndTracking} ...`